dali/pipeline/operator/op_spec.h

```diff
@@ -386,6 +386,18 @@ class DLL_PUBLIC OpSpec {
     return TryGetRepeatedArgumentImpl<T, S>(result, name);
   }
 
+  /**
+   * @brief Checks the Spec for a repeated argument of the given name/type.
+   * Returns the default if an argument with the given name does not exist.
+   */
+  template <typename T, size_t I>
+  DLL_PUBLIC bool TryGetRepeatedArgument(
+      SmallVector<T, I> &result,
+      const string &name) const {
+    using S = argument_storage_t<T>;
+    return TryGetRepeatedArgumentImpl<T, S>(result, name);
+  }
```

Comment:
How about just using `typename Collection` and accepting anything with suitable interface?

Reply:
Done